.github/workflows/deploy-master-staging.yaml

```
@@ -21,6 +21,12 @@ jobs:
       IS_CLOUD_INSTANCE: true
     steps:
       - uses: actions/checkout@v2
+      - name: Set custom version
+        run: |
+          HASH=$(git rev-parse --short HEAD)
+          CURRENT_VERSION=$(cat package.json | jq .version)
```

You can remove the pipe and add the missing `-r` (raw-output):

```suggestion
          CURRENT_VERSION=$(jq -r .version package.json)
```